src/github.ts

```diff
@@ -1,6 +1,6 @@
 import * as github from "@actions/github";
-import * as core from '@actions/core'
-import { IParameter } from './issue'
+import * as core from '@actions/core';
+import { IIssueData } from "./issue";
```

Comment:
This seems like we haven't set up a style linter - we're mixing single and double-quote imports. Maybe set up and use something like `google/gts`?

Actually a lot of changes in this file are style stuff that would be caught with a linter.

Reply:
linter implemented 👍🏻